Makefile

```diff
@@ -47,7 +47,11 @@ db: python
 
 .PHONY: dev
 dev: build/manifest.json python
-	@tox -qe dev
+	@env APP_URL='http://localhost:5000' WEBSOCKET_URL='ws://localhost:5001/ws' supervisord -c conf/supervisord-dev.conf
```

Comment:
It just runs `supervisord` directly, which means you have to have Supervisor installed on your system. Supervisor is added to the docs as a prerequisite.

Based on my experimenting, running either supervisor or honcho inside tox causes them to frequently fail to kill their child processes when tox is killed.

We could still use tox to automate installing supervisor but then just run it directly `@.tox/supervisor/bni/supervisord -c conf/supervisord-dev.conf`. But I don't think we should because:

1. Running a tox command every time `make dev` is run to install, upgrade or downgrade supervisor if necessary, would add a couple of seconds delay every time you run `make dev`

2. We could just check for the presence of a `.tox/supervisor/bni/supervisord` file (which is quick) and only run tox to install it if it's missing, otherwise run `.tox/supervisor/bni/supervisord` directly. But running `.tox/supervisor/bni/supervisord` directly won't upgrade or downgrade supervisor if the supervisor version has changed in the requirements file. If we needed to change the supervisor version we'd then have to ask devs to manually run `tox -e supervisor` _in every project_

I think it's better just to ask people to run `pip install supervisor` once manually. Then they have a single global copy of supervisor that all projects can use. It keeps our `Makefile`'s and `tox.ini`'s simpler. And if we ever need to ask people to upgrade or downgrade supervisor, they only need to run a single `pip` command to do that for all projects.

A limitation is that it means different projects can't use different versions of supervisor, but I don't think that's going to matter. And an individual project could actually install and use its own project copy of supervisor if it ever did happen.